src/aat/resources/features/F-1005 - GlobalSearch - Search cases/S-1005.1.td.json

```diff
@@ -0,0 +1,49 @@
+{
+  "title": "Successfully search for case with one of the new global search parameters",
+  "_guid_": "S-1005.1",
+  "_extends_": "Search_Case_Base",
+  "specs": [
+    "an active profile in CCD",
+    "contains at least one fields from new global search screen",
+    "contains all the mandatory parameters",
+    "has 200 return code"
+  ],
+  "request": {
+    "body": {
+      "searchCriteria": {
+        "caseReferences": [
+          "${[scenarioContext][childContexts][F-1005_CreateCasePreRequisiteCaseworker][testData][actualResponse][body][id]}"
+        ]
+      }
+    }
+  },
+  "expectedResponse": {
+    "_extends_": "Common_200_Response",
```

Comment:
Could create a `Global_Search_200_Response` containing body

```
"resultInfo": {
        "casesReturned": 1,
        "caseStartRecord": 1,
        "moreResultsToGo": false
      }
```
      
which you can then override where needed ?